packages/api/src/command/medical/patient/check-brief-enabled.ts

```diff
@@ -0,0 +1,27 @@
+import { capture, out } from "@metriport/core/util";
+import BadRequestError from "../../../errors/bad-request";
+import { isAiBriefEnabledForCx } from "../../../external/aws/app-config";
+
+export async function checkAiBriefEnabled({
+  cxId,
+  generateAiBrief,
+}: {
+  cxId: string;
+  generateAiBrief: boolean;
+}) {
+  const { log } = out(`AI Brief for cxId: ${cxId}`);
+  const isAiBriefFeatureFlagEnabled = await isAiBriefEnabledForCx(cxId);
+  if (!isAiBriefFeatureFlagEnabled && generateAiBrief) {
```

Comment:
nit: if `generateAiBrief` is false we can return w/o executing the promise `isAiBriefEnabledForCx`